docs/user-guide/pkt-fwd-to-station.md

```diff
@@ -0,0 +1,56 @@
+# Migrate from Packet Forwarder to LoRa Basics Station
+
+Before upgrading to v2.0.0, please take some time to review this document on how to migrate from Packet Forwarder to LoRa Basics Station.
+
+## 1. Creation of a concentrator device
+
+Azure IoT Edge LoRaWAN Starter Kit v2.0.0 support the ability to de-couple the "concentrator" devices from the LoRaWan Network Server, ideally allowing the same LNS to handle concentrators with different antenna configuration or from completely different regions.
+
+Because of this, any specific configuration of the concentrator is now pushed from a new IoT Hub Device representing the "concentrator" device.
+
+Due to the decouplement, more secure authentication modes are also supported for the connection between the concentrator and the LNS.
+
+The concept of provisioning a IoT Hub Device representing the concentrator is explained in the [concentrator provisioning](station-device-provisioning) documentation page.
+
+The supported authentication modes are explained in the [authentication modes](station-authentication-modes) documentation page.
+
+## 2. Connection of the concentrator to the LoRaWan Network Server
+
+After the creation of the concentrator device twin in IoT Hub, it will be possible to connect it to the LoRaWan Network Server.
+
+### Pre-built docker module migration
+
+In case you are using the pre-built [Packet Forwarder module](https://github.com/Azure/iotedge-lorawan-starterkit/blob/116e353bd61133acde13dd9ed6f96ca7156544d1/LoRaEngine/modules/LoRaWanPktFwdModule/start_pktfwd.sh), have a look at the following table for migrating the environment variables to the new [Basic Station module configuration][module-configuration] ones:
```

Comment:
```suggestion
In case you were using the pre-built [Packet Forwarder module](https://github.com/Azure/iotedge-lorawan-starterkit/blob/116e353bd61133acde13dd9ed6f96ca7156544d1/LoRaEngine/modules/LoRaWanPktFwdModule/start_pktfwd.sh), have a look at the following table for migrating the environment variables to the new [Basic Station module configuration][module-configuration] ones:
```